clients/gtest/gemm_gtest.cpp

```diff
@@ -167,11 +164,7 @@ namespace
     // When converted to bool, this functor returns true.
     // Complex is not supported yet.
```

Comment:
I would remove comments about "Complex is not supported yet".

Reply:
understand. update it in commit "update complex change 2".